core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -189,6 +189,22 @@ public static String getStringOrNull(String property, JsonNode node) {
     return getString(property, node);
   }
 
+  public static String getDurationStringOrNull(String property, JsonNode node) {
```

Comment:
nit: for completeness you might also want to add a test to `TestJsonUtil` for this